### PR TITLE
add multiple errors example for ErrorMessage component

### PR DESCRIPTION
**Pull request by kotarella1110**
related: https://github.com/react-hook-form/react-hook-form/issues/1050

### CODE REVIEW

File: src/data/en/api.tsx [hunk 1]
```diff
@@ -1102,6 +1102,12 @@ export default {
               render prop
             </a>{" "}
             for rendering error message or messages.
+            <p>
+              <b className={typographyStyles.note}>Note:</b> you need to set{" "}
+              validateCriteriaMode to 'all' for using <code>messages</code>. If{" "}
+              you don't set the option to 'all', <code>messages</code> will be{" "}
+              <code>undefined</code>.
+            </p>
```
Comment (kotarella1110) [inline src/data/en/api.tsx]: @bluebill1049 please verify if that English sentence is correct.

Comment (bluebill1049) [reply src/data/en/api.tsx]: i think 
```
<b className={typographyStyles.note}>Note:</b> you need to set{" "}
              validateCriteriaMode to 'all' for using <code>messages</code>.
```
will do.

Comment (kotarella1110) [reply src/data/en/api.tsx]: OK! I will fix.

Comment (kotarella1110) [reply src/data/en/api.tsx]: @bluebill1049 completed!

**Comment by bluebill1049** [comment]
awesome merged! thank you

**Pull request merged**